prdoc/pr_3962.prdoc

```diff
@@ -0,0 +1,11 @@
+title: Change fork calculation algorithm.
+
+doc:
+  - audience: Node Dev
+    description: |
+      This PR changes the fork calculation and pruning algorithm to enable future block header pruning.
```

Comment:
Please also mention that forks are now pruned much faster. Best to give some example.